microshift_install/microshift-embed-in-rpm-ostree.adoc

```diff
@@ -6,19 +6,20 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-You can embed {product-title} into a {op-system-ostree-first} {op-system-version} image. Use the content within this section to build a {op-system} image containing {product-title}.
+You can embed {product-title} into a {op-system-ostree-first} {op-system-version} image. Use this guide to build a {op-system} image containing {product-title}.
 
 include::snippets/microshift-tech-preview-snip.adoc[leveloffset=+1]
 
 [id="preparing-for-image-building_{context}"]
 == Preparing for image building
 
-Familiarize yourself with the documentation about https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images[Composing, installing, and managing RHEL for Edge images].
+Read https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images[Composing, installing, and managing RHEL for Edge images].
```

Comment:
Why does this change from 9 to 8?

Reply:
Weird, it shouldn't. We had a problem with a cherry-pick yesterday, the repos were out of sync (extra commit somewhere). Shall fix.